kolibri/plugins/device/assets/src/views/ManageTasksPage/TaskPanel.vue

```diff
@@ -256,10 +256,52 @@
         });
       },
     },
+    watch: {
+      taskPercentage(newPercentage) {
+        if (newPercentage !== null) {
+          const formattedPercentage = (newPercentage * 100).toFixed(2) + '%';
+          const channelName =
+            this.task.extra_metadata.channel_name || this.$tr('unknownChannelName');
+
+          let titlePrefix = '';
+          if (
+            this.task.type === TaskTypes.REMOTECONTENTIMPORT ||
+            this.task.type === TaskTypes.DISKCONTENTIMPORT
+          ) {
+            titlePrefix = this.$tr('importChannelWhole', { channelName });
+          } else if (
+            this.task.type === TaskTypes.DELETECHANNEL ||
+            this.task.type === TaskTypes.DELETECONTENT
+          ) {
+            titlePrefix = this.$tr('deleteChannelWhole', { channelName });
+          }
+
+          document.title = `${formattedPercentage} - ${titlePrefix}`;
+        }
+      },
+      'task.status': {
+        immediate: true,
+        handler(newStatus) {
+          const channelName =
+            this.task.extra_metadata.channel_name || this.$tr('unknownChannelName');
+          if (newStatus === TaskStatuses.CANCELED) {
+            document.title = 'Cancelled - ' + channelName;
```

Comment:
This and other user facing text will need to be translated. A lot of this status text is already referenced elsewhere within this component I believe, so you should be able to find this there.